### PR TITLE
[PLATFORM-955]: Update mockito to ^0.32

### PR DESCRIPTION
**Pull request by MaeIsBad**
https://prima-assicurazioni-spa.myjetbrains.com/youtrack/issue/PLATFORM-955

### CODE REVIEW

File: tests/common/mod.rs [hunk 1]
```diff
@@ -8,166 +9,229 @@ pub fn enable_mockito_logging() {
     env_logger::init();
 }
 
-pub fn create_bridge(status_code: usize, body: &str) -> (Mock, Bridge) {
-    create_bridge_with_path(status_code, body, "/")
+#[async_trait]
+pub trait MockitoServerCreateBridgeExt {
```
Comment (cpiemontese) [inline tests/common/mod.rs:13]: I was looking at the tests but I don't see where this is used :thinking:

**Comment by cpiemontese** [comment]
Great PR! I have just one doubt but I'm ready to approve

File: tests/common/mod.rs (hunk 1 above)
Comment (MaeIsBad) [reply tests/common/mod.rs:13]: It's typically imported and used by the async_bridge rest tests:
https://github.com/primait/bridge.rs/blob/c99d6dd5cd369814b772e7b2add87ce36bd2a01b/tests/async_bridge/rest.rs#L9

Comment (cpiemontese) [reply tests/common/mod.rs:13]: Completely missed that, thanks

**Review by cpiemontese — APPROVED** [review]

**Pull request merged**